MidiBoards/mfmidiboard.schema.json

```diff
@@ -1,172 +1,201 @@
 {
-  "$schema": "http://json-schema.org/draft-06/schema#",
-  "$ref": "#/definitions/MfMidi",
-  "definitions": {
-    "MfMidi": {
-      "type": "object",
-      "additionalProperties": false,
-      "properties": {
-        "$schema": {
-          "type": "string"
-        },
-        "InstanceName": {
-          "type": "string",
-          "description": "Instance name for the device. Required. This is used to match the definition with a connected device."
-        },
-        "DifferingOutputName": {
-          "type": "string",
-          "description": "Name of the midi output port, if different from input name. Optional."
-        },
-        "EncoderNeutralPosition": {
-          "type": "integer",
-          "description": "Set the encoder neutral position. For example 0 or 64 Optional."
-        },
-        "InitialLayer": {
-          "type": "string",
-          "description": "When layer property is used, initial active layer on startup. Optional."
-        },
-        "Inputs": {
-          "type": "array",
-          "description": "List of inputs supported by the device. Required.",
-          "items": {
-            "$ref": "#/definitions/Input"
-          }
-        },
-        "Outputs": {
-          "type": "array",
-          "description": "List of LED outputs supported by the device. Optional.",
-          "items": {
-            "$ref": "#/definitions/Output"
-          }
-        }
+  "$schema": "https://json-schema.org/draft/2019-09/schema#",
+  "$id": "https://mobiflight.com/mfmidiboard.schema.json",
```

Comment:
do we have to actually host those there? would it not be better to host them on GitHub?

Reply:
It's just an ID for the schema, it's not actually hosted there.